### DISCUSSION TITLE
Integrating SVG-Edit in my PHP / HTML / JS application

### DISCUSSION THREAD

**Discussion by Mikeardy** [SVG-Edit/svgedit#905] [General]
Hi foxes,

I'm developing a graphical editor to build HMI screen (they are a kind of industrial drawings representing plants and machineries including valves, pipes, pumps, sensors and so on...).
I'm wondering how to integrate SVG-Edit as the main editor for this purpose.
Let me leave here some points I have in my mind:

- the developing environment is PHP / HTML and JS (no node.js, no frameworks)
- SVG-Edit should be modified to use some basic svg libraries I have (for symbols like valves, pumps and so on...)
- I need an extra characteristic that is a floating window that should have parameters and other data fields according to the graphical object selected by the mouse (like SVG-Edit does right now with graphical object)
- I'm not so advanced programmer, so please be patient.

Could someone try to help me?

Thank you

**Comment by jfhenon** [comment]
Hello,
We use svgEdit in a similar use case. 
- You don't need a framework nor nodejs to use svgedit
-  If you want to change or adapt svgEdit, you would need nodejs
-  You can write an extension for SVGedit to achieve the floating window. 
- You need a few developer skills to achieve your project. I hope you'll find someone on this forum having the bandwidth to support you.

Good luck
